notes.md

```diff
@@ -0,0 +1,19 @@
+Git branch commands
+
+git switch -c <branchname> create a new branch and switch to it
```

Comment:
Wenn man die Commands in Backticks schreibt, also so: `git command`, dann wird einem das in der Markdown-Datei als Code gekennzeichnet. Dieses Zeichen "`" (rechts neben dem ß mit Shift)